docs/dev_guide/contents/map_rotate_custom.rst

```diff
@@ -0,0 +1,90 @@
+.. _map_rotate_custom:
+
+****************************
+Adding a new rotation method
+****************************
+
+If you want to add a new rotation method that can be used by `sunpy.map.GenericMap.rotate`, you can do the following:
+
+.. code-block:: python
+
+    from sunpy.image.transform import add_rotation_function
+
+    @add_rotation_function("my_rotate",
+                        handles_clipping=False, handles_image_nans=False, handles_nan_missing=False)
```

Comment:
Is the indentation really out on these lines like it renders on GH or is GH having a fit?

Reply:
That looks like a rendering on the diff.